custom_functions/find_related_containers.json

```diff
@@ -0,0 +1,78 @@
+{
+    "create_time": "2021-02-02T22:21:31.918867+00:00",
+    "custom_function_id": "bc6cbb96990d43b3df16f31cd43b81f095bd2252",
+    "description": "Takes a provided list of indicator values to search for and finds all related containers. It will produce a list of the related container details.",
+    "draft_mode": false,
+    "inputs": [
+        {
+            "contains_type": [
+                "*"
+            ],
+            "description": "A cef value to search on such as artifact:*.cef.destinationAddress or to search all indicators in container use: *",
```

Comment:
I think this is a bit confusing because usually we give an example value that is the literal value being passed in, but your example is the data path which will get converted to a literal value before running. It might be a bit more clear if it said

> An indicator value to search on, such as a file hash or IP address. To search on all indicator values in the container, use "*".

Reply:
Made adjustments to description